pkgs/development/python-modules/torch/default.nix

```diff
@@ -4,6 +4,10 @@
   MPISupport ? false, mpi,
   buildDocs ? false,
 
+  # tests.cudaAvailable:
+  callPackage,
+  torch,
```

Comment:
No, I'm not sure we want that. If anything, we may want to take specifically `torchWithCuda` from the package set instead of `torch`. It's the same for all other passthru tests that are based on overriding the parent derivation with `doCheck`: they really test the instance from the package set, even if you override it locally. If you want to test the overridden version, I think you have to overlay the package set.

Besides, I'm pretty sure `self = buildPythonpackage` will actually fail at `overridePythonAttrs`. Maybe `finalAttrs` could work, if we really wanted to make this happen. Then we should do the same thing to the rest of nixpkgs:)